pages/tests/[testsByRunId].tsx

```diff
@@ -303,21 +311,7 @@ function Tests({
                       }}
                     >
                       {" "}
-                      {test_run.test_run_data &&
-                      test_run.test_run_data.spectre_test_run_url ? (
-                        <div>
-                          <Button
-                            href={
-                              test_run.test_run_data.spectre_test_run_url +
-                              "?status=fail"
-                            }
-                            className={classes.spectreButton}
-                            target="_blank"
-                          >
-                            Spectre
-                          </Button>
-                        </div>
-                      ) : null}
+                      <SmartLinksTestRun project_id={test_run.project_id} environment={test_run.environment} test_run_id={test_run.test_run_id}/>
```

Comment:
let's have a quick run through on how to set up a spectre link.
Or better a tooltip for the user to understand this functionality.